faunadb-java-dsl/src/main/java/com/faunadb/client/query/Language.java

```diff
@@ -354,1261 +484,2003 @@ public static Expr Value(HighPrecisionTime value) {
   }
 
   /**
-   * Creates a new Date value.
+   * Encodes the given {@link LocalDate} as an {@link Expr} instance.
    *
+   * @param value the date to be encoded
+   * @return a new {@link Expr} instance
    * @see <a href="https://fauna.com/documentation/queries#values">FaunaDB Values</a>
+   * @see LocalDate
    */
   public static Expr Value(LocalDate value) {
     return new DateV(value);
   }
 
   /**
-   * Creates a new Bytes value.
+   * Encodes the given {@link Byte} array as an {@link Expr} instance.
    *
+   * @param bytes the byte array to be encoded
+   * @return a new {@link Expr} instance
    * @see <a href="https://fauna.com/documentation/queries#values">FaunaDB Values</a>
    */
   public static Expr Value(byte[] bytes) {
     return new BytesV(bytes);
   }
 
   /**
-   * Creates a null value.
+   * Creates a new expression representing a null value.
    *
+   * @return a new {@link Expr} instance
    * @see <a href="https://fauna.com/documentation/queries#values">FaunaDB Values</a>
    */
   public static Expr Null() {
     return NullV.NULL;
   }
 
   /**
-   * Creates a new Object value wrapping the provided map.
+   * Creates a new object wrapping the {@link Map} provided.
+   * For convenience, the following helpers are available:
+   * <ul>
+   *   <li>{@link #Obj()}</li>
+   *   <li>{@link #Obj(String, Expr)}</li>
+   *   <li>{@link #Obj(String, Expr, String, Expr)}</li>
+   *   <li>{@link #Obj(String, Expr, String, Expr, String, Expr)}</li>
+   *   <li>{@link #Obj(String, Expr, String, Expr, String, Expr, String, Expr)}</li>
+   *   <li>{@link #Obj(String, Expr, String, Expr, String, Expr, String, Expr, String, Expr)}</li>
+   * </ul>
    *
+   * @param values the key/value {@link Map} to be wrapped
+   * @return a new {@link Expr} instance
    * @see <a href="https://fauna.com/documentation/queries#values">FaunaDB Values</a>
    */
   public static Expr Obj(Map<String, ? extends Expr> values) {
     return Fn.apply("object", Fn.apply(values));
   }
 
   /**
-   * Creates an empty Object value.
+   * Creates an empty object.
    *
+   * @return a new {@link Expr} instance
    * @see <a href="https://fauna.com/documentation/queries#values">FaunaDB Values</a>
    */
   public static Expr Obj() {
     return Obj(ImmutableMap.<String, Expr>of());
   }
 
   /**
-   * Creates a new Object value with the provided entries.
+   * Creates a new object with the provided key and value.
    *
+   * @param k1 a key
+   * @param v1 a value
+   * @return a new {@link Expr} instance
    * @see <a href="https://fauna.com/documentation/queries#values">FaunaDB Values</a>
    */
   public static Expr Obj(String k1, Expr v1) {
     return Obj(ImmutableMap.of(k1, v1));
   }
 
   /**
-   * Creates a new Object value with the provided entries.
+   * Creates a new object with the provided key/value pairs.
```

Comment:
It feels like we should be precise, as the function takes a specific number of key/value pairs
`Creates a new object with the provided key/value pairs`
change to 
`Creates a new object with two key/value pairs`